### ISSUE TITLE
cert-manager fails to install using Helm v2.12.0 for HA installation

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#17002]
**What kind of request is this (question/bug/enhancement/feature request):**
bug (upstream)

**Steps to reproduce (least amount of steps as possible):**
As described on https://rancher.com/docs/rancher/v2.x/en/installation/ha/helm-rancher/, `cert-manager` needs to be installed for certain SSL configurations.

Running the described command using Helm v2.12.0 fails:

```
$ helm install stable/cert-manager   --name cert-manager   --namespace kube-system
Error: release cert-manager failed: customresourcedefinitions.apiextensions.k8s.io "certificates.certmanager.k8s.io" already exists
```

Workaround: Use Helm v2.11.0

Ref https://github.com/jetstack/cert-manager/issues/1144

**Comment by davidnuzik** [comment]
Another workaround that worked for me is installing older cert-manager (although may only be ideal for testing purposes)
`helm install --name cert-manager --namespace kube-system --version v0.4.1 stable/cert-manager`
The referenced issue mentioned it

**Comment by superseb** [comment]
This should be fixed by using v2.12.1: https://github.com/helm/helm/releases/tag/v2.12.1
